library/flink

```diff
@@ -3,17 +3,32 @@
 Maintainers: The Apache Flink Project <dev@flink.apache.org> (@ApacheFlink)
```

Comment:
As a note to take back for a future PR, `@ApacheFlink` here is supposed to be a GitHub handle, and this isn't -- it would be _ideal_ if we could get a list of human beings that we can actually @-notify on PRs here, but at the very least this should be updated to be a real GitHub handle.